thirdparty_containers/lucene-solr/playlist.xml

```diff
@@ -15,7 +15,7 @@
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
 	<test>
 		<testCaseName>lucene_solr_nightly_smoketest</testCaseName>
-	<command>docker run --rm -v $(JDK_HOME):/java -v $(REPORTDIR)/junitreports/lucene-solr:/lucene-solr/lucene/build/core/test adoptopenjdk-lucene-solr-test:latest ; \
+	<command>docker run --rm -v $(JDK_HOME):/java -v $(REPORTDIR)/junitreports/:/lucene-solr/lucene/build/core/test adoptopenjdk-lucene-solr-test:latest ; \
```

Comment:
You could try remove the junitreports dir from this command, to dump reports directly into test_output_xxxx dir (REPORTDIR)